Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml

```diff
@@ -390,3 +461,5 @@ outputs: []
 tests:
```

Comment:
Could not add a comment on Line 458, so adding it here. Update the wording on the description as "The security group that will be used for remediating internet exposures. "

Reply:
updated in https://github.com/demisto/content/pull/24786/commits/2c114e3be9e63b53f19e650414257b44b2aa54b0